docs/pages/guides/monorepos.mdx

```diff
@@ -84,8 +88,10 @@ config.resolver.nodeModulesPaths = [
   path.resolve(projectRoot, 'node_modules'),
   path.resolve(workspaceRoot, 'node_modules'),
 ];
+/* @info Starting from Expo SDK 50, you no longer should disable the hierarchical lookup by default. Learn more in the section below. */
```

Comment:
I think this should part of the main point above rather than having a highlight here in the code snippet (it's easier to miss this). We can also link the right section from the main context.

Reply:
Ok, so the reason why I put it in here, is for two things:

1. We don't emphasize anymore on "disable hierarchical lookup", so I'm not sure how well it reads when you start the main point with "Do NOT do this" (as opposed to "do this").
2. Since we _previously_ did recommend this, I kept it in the same spot, but disabled with more context on _why_ it's disabled.

If you think that's not good, I'm open for suggestions 😄